apps/innbyggers-flate/src/pages/TiltakPage.tsx

```diff
@@ -1,3 +1,220 @@
+import { useDeltakerContext } from '../DeltakerContext.tsx'
+import {
+  BodyLong,
+  BodyShort,
+  ExpansionCard,
+  Heading,
+  HStack,
+  Label,
+  Link,
+  LinkPanel,
+  List
+} from '@navikt/ds-react'
+import {
+  EMDASH,
+  formatDateFromString,
+  getDeltakerStatusAarsakText,
+  hentTiltakNavnHosArrangørTekst,
+  INNHOLD_TYPE_ANNET
+} from '../utils/utils.ts'
+import {
+  DeltakerResponse,
+  DeltakerStatusType,
+  Tiltakstype
+} from '../api/data/deltaker.ts'
+import { ChatElipsisIcon, ChevronRightIcon } from '@navikt/aksel-icons'
+import { getDialogUrl } from '../utils/environment-utils.ts'
+import { DeltakerIStatusTag } from '../components/DeltakerIStatusTag.tsx'
+import { HvaErDette } from '../components/HvaErDette.tsx'
+
+const skalViseDeltakelsesmengde = (deltaker: DeltakerResponse) => {
+  return (
+    deltaker.deltakerliste.tiltakstype == Tiltakstype.ARBFORB ||
+    deltaker.deltakerliste.tiltakstype == Tiltakstype.VASV
+  )
+}
+
+const deltakelsesmengdeString = (deltaker: DeltakerResponse): string => {
+  if (deltaker.dagerPerUke !== null) {
+    if (deltaker.dagerPerUke === 1) {
+      return `${deltaker.deltakelsesprosent ?? 100}\u00A0% ${deltaker.dagerPerUke} dag i uka`
+    } else {
+      return `${deltaker.deltakelsesprosent ?? 100}\u00A0% ${deltaker.dagerPerUke} dager i uka`
+    }
+  }
+
+  return `${deltaker.deltakelsesprosent}\u00A0%`
+}
+
 export const TiltakPage = () => {
-  return <div className="flex flex-col items-start">Tiltaksiden</div>
+  const { deltaker } = useDeltakerContext()
+
+  const tiltakOgStedTekst = hentTiltakNavnHosArrangørTekst(
+    deltaker.deltakerliste.tiltakstype,
+    deltaker.deltakerliste.arrangorNavn
+  )
+  const skalViseDato =
+    deltaker.status.type !== DeltakerStatusType.IKKE_AKTUELL &&
+    deltaker.status.type !== DeltakerStatusType.AVBRUTT_UTKAST
+  const bakgrunsinformasjon =
+    deltaker.bakgrunnsinformasjon && deltaker.bakgrunnsinformasjon.length > 0
+      ? deltaker.bakgrunnsinformasjon
+      : EMDASH
+
+  let dato = EMDASH
+  if (
+    deltaker.startdato &&
+    deltaker.sluttdato &&
+    deltaker.startdato !== EMDASH &&
+    deltaker.sluttdato !== EMDASH
+  ) {
+    dato = `${formatDateFromString(deltaker.startdato)} - ${formatDateFromString(deltaker.sluttdato)}`
+  } else if (deltaker.startdato && deltaker.startdato !== EMDASH) {
+    dato = formatDateFromString(deltaker.startdato)
+  }
+
+  return (
+    <div className={'bg-white px-12 py-4 w-full'}>
```

Comment:
Trur vi og kan ta vekk `px-12` også, blir en stor padding i x-retininga, vi kan sikkert behold py-4 for nå så kan Linn se på det, vi må ha en tilbakeknapp som jeg er litt usikker på per nå også